lib/common/client/middleware/response/mhv_xml_html_errors.rb

```diff
@@ -5,85 +5,26 @@ module Middleware
       module Response
         class MhvXmlHtmlErrors < Faraday::Response::Middleware
           include SentryLogging
-
-          ERROR_LIST = %w(service_outage generic_xml generic).freeze
+          attr_reader :status
 
           def on_complete(env)
-            return unless error_and_xml_or_html?(env)
-
-            @doc = html?(env) ? Nokogiri::HTML(env.body) : Nokogiri::XML(env.body)
-
-            jsoned_error = ERROR_LIST.each_with_object('') { |error, je| break je if (je = send(error)) }
-            jsoned_error['errorCode'] = env.status if jsoned_error['errorCode'].blank?
+            return if env.success?
+            return unless env.response_headers['content-type'] =~ /\b(xml|html)/
+            @status = env.status.to_i
 
-            env.body = jsoned_error
-            env.response_headers['content-type'] = 'application/json'
+            extra_context = { original_status: @status, original_body: env.body.delete('%') }
+            log_message_to_sentry('Could not parse XML/HTML', :warn, extra_context)
```

Comment:
Maybe make this message a bit more specific "Could not parse XML/HTML response from MHV".